### ISSUE TITLE
Support datapackage spec

### ISSUE THREAD

**Issue by Hypercubed** [chiasm-project/chiasm-dataset#10]
Hello Curran,

Me again!  I am currently extracting the portion of [Project-Chi](https://github.com/Hypercubed/Project-Chi) that handles the data processing.  WIP is [here](https://github.com/Hypercubed/chi-datapackage/tree/dev).  One big difference between the Project-χ approach and chiasm-dataset is that I am trying to follow the [Data Packages sepc](http://specs.frictionlessdata.io/data-packages/).  Have you considered this for Chiasm.  It would be nice to avoid duplication of efforts.

**Comment by Hypercubed** [comment]
Have you thought about this?  I have made some progress here: https://github.com/Hypercubed/chi-datapackage

(Note, I'm using the "chi" prefix for project-chi).

**Comment by curran** [comment]
Hi Jayson,

Thank you for referring me to Data Packages. It is something I am not familiar with, and it looks like they are doing a great job with it. I'd like to spend some more time learning about it. Maybe it is something we can use.

I was looking to see if the spec supports columns that represent intervals (e.g. time intervals, or numeric intervals like age ranges). Do you know if the Data Packages spec supports those?

**Comment by Hypercubed** [comment]
Yes, you need to look at the Table Schema Spec.  It specifies support for ISO 8601 durations.  http://specs.frictionlessdata.io/json-table-schema/#duration
